tests/simplex/q_witherden_vincent_01.cc

```diff
@@ -0,0 +1,82 @@
+#include <deal.II/base/function_lib.h>
+
+#include <deal.II/simplex/quadrature_lib.h>
+
+#include "../tests.h"
+
+template <int dim>
+void
+print(unsigned int n_points_1D)
```

Comment:
```suggestion
print(const unsigned int n_points_1D)
```
Same in a couple more places below